third_party/proto/buf.yaml

```diff
@@ -30,4 +30,4 @@ lint:
     - confio
     - gogoproto
     - google
-    - tendermint
+    - tendermint
```

Comment:
I think appeared as a result of #781 where I made the changes here and reverted them. Not sure why this is flagged by the GH diff